Makefile

```diff
@@ -15,4 +15,7 @@ python_venv:
  		./quickstart_venv/bin/pip3 install -r python/requirements.txt && \
  		set -a && source .env && set +a && ./quickstart_venv/bin/python3 -m python.src.server
 
-
+ruby_local:
```

Comment:
Should we just call it ruby and python_venv - python? Do you plan to add docker version for Ruby?

Reply:
we can definitely change those names. A `ruby_docker` is in the works but there's a bug with it I'm trying to work out.

Reply:
turns out you can't simply name them `python` `ruby` or `node`. Makes sense because when you run `make python` it'll try and make the actual python in the system as opposed to the `python` task in the makefile. I've renamed them to be consistent however. Also I still haven't figured out the docker issue for ruby, but I don't think that should stop us from merging this PR.